addon/-private/formatters/format-message.ts

```diff
@@ -12,7 +12,7 @@ import type { PrimitiveType } from 'intl-messageformat';
 import Formatter from './-base';
 const {
   Handlebars: {
-    // @ts-expect-error Upstream types are incomplete.
+    // @ts-ignore: Upstream types are incomplete.
```

Comment:
When I had tested `ember-intl@6.1.1` in [`ember-workshop`](https://github.com/ijlee2/ember-workshop), I found the need to patch this file by changing `@ts-expect-error` to `@ts-ignore`. (In essence, I needed to revert https://github.com/ember-intl/ember-intl/pull/1776/commits/4fec508bc57bb790e2c7d44f538b4c2d8bf19647.)

It seems that, depending on how TypeScript is configured on the the end-user's project side, the expected errors may not occur. At the time of writing, `ember-workshop` makes use of `@tsconfig/ember@3.0.1`.